dev/CommandBarFlyout/Strings/en-us/Resources.resw

```diff
@@ -117,6 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="CommandBarFlyoutCommandBarLocalizedControlType.Value" xml:space="preserve">
+    <value>menu localized</value>
```

Comment:
>localized [](http://example.com/codeflow?start=16&length=9)

should this include 'localized' as well ?

Reply:
Oops, good catch.  I had that there so it was easy to spot that we were pulling in the localized string, but forgot to revert that.  Will fix.